extension-manifest-v3/src/background/telemetry/index.js

```diff
@@ -113,9 +114,16 @@ chrome.runtime.onMessage.addListener((msg) => {
     log('Telemetry recordUTMs() error', error);
   }
 
-  if (JUST_INSTALLED) {
-    telemetry.ping('install');
-  }
-  telemetry.ping('active');
+  observe('terms', async (terms) => {
+    telemetryEnabled = terms;
```

Comment:
This function runs at least once every background run. You don't need a flag, just check terms and call pings.

Reply:
telemetry is always there, as it is used to set the uninstall url